packages/utilities/src/customizations/customizable.server.test.tsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * @jest-environment node
+ */
+import * as React from 'react';
```

Comment:
decoupled SSR jest test that were causing issues within browser mocked env (react loaded incompatible modules)